### PR TITLE
RevEng: Adding Force flag to prevent existing files being overwritten by mistake

### PR DESCRIPTION
**Pull request by lajones**
Fix for #3856.

### CODE REVIEW

File: src/EntityFramework.Commands/Commands/Program.cs [hunk 1]
```diff
@@ -655,13 +660,15 @@ public virtual int RemoveMigration([CanBeNull] string context)
                 [CanBeNull] List<string> schemaFilters,
                 [CanBeNull] List<string> tableFilters,
                 bool useDataAnnotations,
+                bool force,
```
Comment (natemcmaster) [inline src/EntityFramework.Commands/Commands/Program.cs]: "force" is good as a command line parameter but in code its meaning is obscure. "overwriteFiles" would be more clear.

Comment (lajones) [inline src/EntityFramework.Commands/Commands/Program.cs]: Sounds good. Will fix.

**Comment by natemcmaster** [comment]
:shipit: w/ feedback updates

**Comment by lajones** [comment]
Checked in with updates - commit c698d65.

**Pull request closed without merging**